### PR TITLE
RecyclerView Item Holder View is accessable in code after OnBind...

### PR DESCRIPTION
**Pull request by thefex**
Fixes issue #1566

### CODE REVIEW

File: MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/IMvxRecyclerAdapterBindableHolder.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+using System;
+using MvvmCross.Droid.Support.V7.RecyclerView.Model;
+
+namespace MvvmCross.Droid.Support.V7.RecyclerView
+{
+    public interface IMvxRecyclerAdapterBindableHolder
+    {
+        event Action<MvxViewHolderBindedEventArgs> MvxViewHolderBinded;
```
Comment (kjeremy) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/IMvxRecyclerAdapterBindableHolder.cs]: `Binded` is weird.  I think it needs to be something with `Bound` in it (and drop the `Mvx` bit) like `ViewHolderBound` or something.

File: MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Model/MvxViewHolderBindedEventArgs.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+namespace MvvmCross.Droid.Support.V7.RecyclerView.Model
+{
+    public class MvxViewHolderBindedEventArgs
```
Comment (kjeremy) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Model/MvxViewHolderBindedEventArgs.cs]: Same here with `Binded` vs `Bound`

File: MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Model/MvxViewHolderBindedEventArgs.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            ItemPosition = itemPosition;
+            DataContext = dataContext;
+            Holder = holder;
+        }
+
+        public int ItemPosition { get; }
+
+        public object DataContext { get; }
+
+        public Android.Support.V7.Widget.RecyclerView.ViewHolder Holder { get; }
```
Comment (kjeremy) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Model/MvxViewHolderBindedEventArgs.cs]: Is it the `ViewHolder` that you need access to or the `View`?

Comment (thefex) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Model/MvxViewHolderBindedEventArgs.cs]: ViewHolder constains all required informations like:
ItemView,
Item position,
ItemView Type.
I guess it's better to expose that than just pure View.

File: MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/IMvxRecyclerAdapterBindableHolder.cs (hunk 1 above)
Comment (thefex) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/IMvxRecyclerAdapterBindableHolder.cs]: Right, gonna change that.

**Pull request merged**